### PR TITLE
Add `jupyterlab-topbar-text`

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jtpio** [comment]
cc @conda-forge/help-python this is ready for review. Thanks!

File: recipes/jupyterlab-topbar-text/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+about:
+  home: https://github.com/jupyterlab-contrib/jupyterlab-topbar-text
+  summary: JupyterLab extension to display arbitrary text in the top bar
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - fcollonval
```
Comment (jtpio) [inline recipes/jupyterlab-topbar-text/meta.yaml:45]: cc @fcollonval adding you here since the repo is part of the `jupyterlab-contrib` org.

Comment (fcollonval) [reply recipes/jupyterlab-topbar-text/meta.yaml:45]: Thanks @jtpio
i agree to be maintainer.

**Comment by jtpio** [comment]
@conda-forge/help-python @conda-forge/staged-recipes this should be good to go, thanks!

**Pull request merged**